ui_framework/src/services/index.js

```diff
@@ -6,6 +10,5 @@ export {
 } from './accessibility';
```

Comment:
Good point. Honestly I just refactored the `key_codes` file and left the `accessible_click_keys` as it was. I fixed that with now with another commit. Thanks.